Parse/Parse/Internal/Commands/CommandRunner/URLSession/Session/PFURLSession.m

```diff
@@ -257,4 +263,19 @@ - (void)URLSession:(NSURLSession *)session
     completionHandler(nil); // Prevent any caching for security reasons
 }
 
+///--------------------------------------
+#pragma mark - NSURLSessionDelegate
+///--------------------------------------
+
+- (void)URLSession:(NSURLSession *)session
+didReceiveChallenge:(NSURLAuthenticationChallenge *)challenge
+ completionHandler:(void (^)(NSURLSessionAuthChallengeDisposition, NSURLCredential * _Nullable))completionHandler {
+    if (_urlSessionChallengeDelegate != NULL) {
+        [_urlSessionChallengeDelegate
+         URLSession:session
+         didReceiveChallenge:challenge
+         completionHandler:completionHandler];
+    }
```

Comment:
Be sure to implement the `else` here just in case the delegate is `NULL` or else all tls connections could be blocked unintentionally. To see what the `else` needs to pass to the `completionHandler ` you can checkout the ParseSwift implementation: https://github.com/parse-community/Parse-Swift/blob/1db06743febe41708385d672271a8e51d2ecafe4/Sources/ParseSwift/API/ParseURLSessionDelegate.swift#L38-L45

Reply:
That's a good point, thanks.
I was looking into your Swift version. Looking great.